front/src/applications/stdcmV2/components/StdcmConfig.tsx

```diff
@@ -0,0 +1,125 @@
+
+  const dispatch = useAppDispatch();
+  const { updateGridMarginAfter, updateGridMarginBefore, updateStdcmStandardAllowance } =
+    useOsrdConfActions() as StdcmConfSliceActions;
+
+  const { pathfindingState } = usePathfindingV2({
+    pathProperties,
+    setPathProperties,
+  });
+
+  useEffect(() => {
+    if (isPending) {
+      loaderRef?.current?.scrollIntoView({ behavior: 'smooth' });
+    }
+  }, [isPending]);
+
+  useEffect(() => {
+    dispatch(updateGridMarginAfter(35));
+    dispatch(updateGridMarginBefore(35));
+    dispatch(updateStdcmStandardAllowance({ type: 'time_per_distance', value: 4.5 }));
+  }, []);
+
+  return (
+    <div className="stdcm-v2__body">
+      <div className="stdcm-v2-simulation-settings">
+        <div className="stdcm-v2-consist-container">
+          <StdcmConsist
+            disabled={isPending}
+            setCurrentSimulationInputs={setCurrentSimulationInputs}
+          />
+        </div>
+        <div className="stdcm-v2__separator" />
+        <div className="stdcm-v2-simulation-itinerary">
+          {/* //TODO: rename StdcmDefaultCard */}
+          {/* <StdcmDefaultCard text="Indiquer le sillon antérieur" Icon={<ArrowUp size="lg" />} /> */}
+          <StdcmOrigin
+            disabled={isPending}
+            setCurrentSimulationInputs={setCurrentSimulationInputs}
+          />
+          <StdcmVias disabled={isPending} setCurrentSimulationInputs={setCurrentSimulationInputs} />
+          <StdcmDestination
+            disabled={isPending}
+            setCurrentSimulationInputs={setCurrentSimulationInputs}
+          />
+          {/* <StdcmDefaultCard text="Indiquer le sillon postérieur" Icon={<ArrowDown size="lg" />} /> */}
+          {/* //TODO: replace .wizz-effect once we have the definitive one */}
+          <div
+            className={cx('stdcm-v2-launch-request', {
+              'wizz-effect': !pathfindingState.done,
+              'pb-5': !pathfindingState.error,
+            })}
+          >
+            {currentSimulationInputs && (
+              <Button
+                label={t('simulation.getSimulation')}
+                onClick={() => {
+                  if (pathfindingState.done) {
+                    launchStdcmRequest();
+                    setCurrentSimulationInputs(undefined);
+                  }
+                }}
+              />
+            )}
+            {pathfindingState.error && (
+              <div className="warning-box">
+                <span>
+                  <Alert variant="fill" />
+                </span>
+                <p className="mb-0">{t('pathfindingFailed')}</p>
+              </div>
+            )}
+          </div>
+          {isPending && <StdcmLoader cancelStdcmRequest={cancelStdcmRequest} ref={loaderRef} />}
+        </div>
+      </div>
+      <div className="osrd-config-item-container osrd-config-item-container-map stdcm-v2-map">
+        <Map hideAttribution />
```

Comment:
I know it hasn't been refined yet for the new map but as long as we keep the old map, maybe we can still display the path on it.

```suggestion
        <Map hideAttribution pathProperties={pathProperties} />
```

Reply:
On the issue, it has been decided to not display the path on the map